server/sockets/join.js

```diff
@@ -19,16 +20,28 @@ const {
  *
  * @param {Server} io
  */
+
 const matchMaker = async (io) => {
   while (getWaitingUserLen() > 1) {
-    const chat = await createChat(getRandomPairFromWaitingList());
+    const [user1, user2] = getRandomPairFromWaitingList();
```

Comment:
Please use a more proper naming convention, avoid using numbers, so better to be userOne and userTwo